controls/pcidss_4_ocp4.yml

```diff
@@ -1334,18 +1332,41 @@ controls:
         malware.
       levels:
         - base
-      status: pending
+      status: supported
```

Comment:
Can you refresh me on the difference between `supported` and `partial` in this context?

Reply:
https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status

partial: While work has been done to address this control, there is still work needed to fully address it.

supported: The control is addressed by the product (but is missing content automation).